misk-eventrouter/src/main/kotlin/misk/eventrouter/ConsistentHashing.kt

```diff
@@ -6,11 +6,14 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class ConsistentHashing @Inject constructor(
-  private val hashFunction: HashFunction = Hashing.murmur3_32(),
-  private val mod: Long = 65536L,
-  private val virtualPoints: Int = 16
+class ConsistentHashing(
+  private val hashFunction: HashFunction,
+  private val mod: Long,
+  private val virtualPoints: Int
 ) : ClusterMapper {
+
+  @Inject constructor() : this(Hashing.murmur3_32(), 65536L, 16)
```

Comment:
This seemed unrelated, but Guice was throwing the following error because the constructor, having all args with default values, resulted in both a no-arg constructor **and** an all arg constructor being generated with the `@Inject` annotation.  

```
1) misk.eventrouter.ConsistentHashing has more than one constructor annotated with @Inject. Classes must have either one (and only one) constructor annotated with @Inject or a zero-argument constructor that is not private.
  at misk.eventrouter.ConsistentHashing.class(ConsistentHashing.kt:9)
  at misk.eventrouter.RealEventRouterModule.configure(RealEventRouterModule.kt:93)
```

I'm not sure if there's a better way to solve this, like a way to tell the compiler which generated constructor you want the annotation to **actually** be on, so this is the best I've got.